src/libs/actions/Policy/Policy.ts

```diff
@@ -3050,6 +3051,21 @@ function inviteWorkspaceEmployeesToUber(policyID: string, emails: string[]) {
     API.write(WRITE_COMMANDS.INVITE_WORKSPACE_EMPLOYEES_TO_UBER, params, {optimisticData, successData, failureData});
 }
 
+/**
+ * Clears an error for a specific Uber employee
+ */
+function clearUberEmployeeError(policyID: string, email: string) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {
+        receiptPartners: {
+            uber: {
+                employees: {
+                    [email]: null,
```

Comment:
This would clear all the data, not just the error, no?

Reply:
It needs to go back to the same state it was before the failure, no?

Reply:
It was? Don't we use this for resending invitations too?
In any case if this is correct, the name is not since the method is not clearing the error in an uber employee, it's clearing the whole employee